### PR TITLE
New example added

### PR DESCRIPTION
**Pull request by dvoineu**

### CODE REVIEW

File: Source/EvolvExecution.swift [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        let allocations = EvolvAllocations(rawAllocations, store: store)
+        let node = try allocations.value(forKey: key, participant: participant)
+        
+        guard node.type == defaultValue.type else {
+            throw Error.mismatchTypes
+        }
+        
+        let activeExperiements = allocations.getActiveExperiments()
+        
+        if alreadyExecuted.isEmpty || alreadyExecuted != activeExperiements {
```
Comment (robertsevernsentient) [inline Source/EvolvExecution.swift:68]: Not quite following what this is doing -- I'm not sure that the javascript SDk has this concept?  This is from the previous SDK again

File: Source/EvolvParticipant.swift [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+public class EvolvParticipantBuilder: NSObject {
+    
+    private var userId: String
+    private var sessionId: String
+    private var userAttributes: [String: String]
+    
+    override init() {
+        self.userId = UUID().uuidString
```
Comment (robertsevernsentient) [inline Source/EvolvParticipant.swift:47]: The application provide the user id, we should not be generating it

File: Source/EvolvParticipant.swift [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        self.sessionId = UUID().uuidString
```
Comment (robertsevernsentient) [inline Source/EvolvParticipant.swift:48]: session id is not going to be used

File: Source/EvolvParticipant.swift [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        self.userAttributes = [
```
Comment (robertsevernsentient) [inline Source/EvolvParticipant.swift:49]: This is an idea from the old SDK -- the equivalent in the javascript SDK is the context

File: Source/EvolvRawAllocation.swift [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+import Foundation
+
+/// The set of configurations that have been given to the participant, the values that are being
+/// experimented against
+public class EvolvRawAllocation: NSObject, Decodable {
+    
+    struct State: OptionSet {
+        let rawValue: Int
+        
+        static let touched = State(rawValue: 1 << 0)
```
Comment (robertsevernsentient) [inline Source/EvolvRawAllocation.swift:28]: Handled by context

**Review by robertsevernsentient — COMMENTED** [review]
This seems to be 95% copy pasted from https://github.com/evolv-ai/EvolvKit
See the comments for specifics - but the reason we're porting over the javascript sdk is that some fundamental concepts have changed since the previous native SDKs were written

**Comment by dvoineu** [comment]
Started new version of SDK from scratch.

**Pull request closed without merging**